cfme/tests/candu/test_graph_groupbytag.py

```diff
@@ -61,21 +71,42 @@ def test_tagwise(candu_db_restore, interval, graph_type, gp_by, host):
         initialEstimate: 1/4h
         casecomponent: CandU
     """
-    view = navigate_to(host, 'candu')
+    if entity == 'host':
+        view = navigate_to(entity_object, 'candu')
+    elif entity == 'cluster':
+        view = navigate_to(entity_object, 'Utilization')
```

Comment:
aaaaaahhh this was my bad :). I think we need to use same name. TODO: need PR for this.